libs/gltfio/src/AssetLoader.cpp

```diff
@@ -444,22 +444,20 @@ void FAssetLoader::createRenderable(const cgltf_data* srcAsset, const cgltf_node
 
     Aabb aabb;
 
-    // glTF spec says that all primitives MUST have the same number of morph targets in the same order.
-    const cgltf_size numMorphTargets = mesh->weights_count;
-    builder.morphing(numMorphTargets);
-
+    cgltf_size numMorphTargets = 0;
     // For each prim, create a Filament VertexBuffer, IndexBuffer, and MaterialInstance.
     for (cgltf_size index = 0; index < nprims; ++index, ++outputPrim, ++inputPrim) {
         RenderableManager::PrimitiveType primType;
         if (!getPrimitiveType(inputPrim->type, &primType)) {
             slog.e << "Unsupported primitive type in " << name << io::endl;
         }
 
-        if (numMorphTargets > 0 && inputPrim->targets_count != numMorphTargets) {
-            slog.e << "Sister primitives must all have the same number of morph targets."
-                   << io::endl;
-            mError = true;
-            continue;
+        if (inputPrim->targets_count > 0) {
+            if (numMorphTargets > 0 && inputPrim->targets_count != numMorphTargets) {
+                slog.e << "Sister primitives must all have the same number of morph targets."
+                       << io::endl;
+            }
```

Comment:
Should we still set `mError = true` here?

Reply:
Actually I just revert changes. Your suggestion will be great. I will update this.